starters/adapters/aws-lambda/adapters/aws-lambda/vite.config.ts

```diff
@@ -0,0 +1,25 @@
+import { nodeServerAdapter } from '@builder.io/qwik-city/adapters/node-server/vite';
+import { extendConfig } from '@builder.io/qwik-city/vite';
+import baseConfig from '../../vite.config';
+import { writeFileSync } from 'node:fs';
+import { join } from 'node:path';
+
+export const entryPointWithDot = () => ({
```

Comment:
It's because it is necessary for a filename without dots. 

https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html?icmpid=docs_lambda_help